templates/articles/translations/create.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+
+{% block body %}
+<h1>Creating a translation in {{ language }} for article “{{ article.title }}”</h1>
+
+<form method=post action=>
```

Comment:
Is `action=` with no value valid, without even a dot?

Reply:
My browser looks to appreciate it. This way it keeps the current query string!

Reply:
I break the w3c validator. although `action=.` should do the job :)
And [I had this DoD in mind](https://github.com/cafebabel/cafebabel.com/wiki/DoD#pour-le-site) (ok ok, for that point ^^ )

Reply:
Done 😘